### DISCUSSION TITLE
Add state_class to entities to allow long-term statistics to be collected

### DISCUSSION THREAD

**Discussion by Vertikar** [briis/hass-weatherflow2mqtt#80] [Ideas]
Would be great to be able to have the new long-term stats feature ( https://www.home-assistant.io/blog/2021/09/01/release-20219/#long-term-statistics-unlocked-for-all-sensors ) available for the weather data from weatherflow - dev details here: https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

I can see zigbee2mqtt is already exposing this, looks like it'd be added to the autodiscovery.

**Comment by briis** [comment]
Great idea. I will add this as an enhancement in the Issues section. That will also take away the need for doing our own min/max sensors. Thanks
